packages/neuron-ui/src/widgets/DatetimePicker/index.tsx

```diff
@@ -155,14 +128,7 @@ const DatetimePicker = ({
             onKeyPress={onKeyPress}
           />
         )}
-        <Calendar
-          value={selected}
-          minDate={new Date()}
-          onChange={onCalendarChange}
-          inline
-          locale="es"
-          className={styles.calendar}
-        />
+        <Calendar value={selected} minDate={tomorrow} onChange={onCalendarChange} />
```

Comment:
I specifically designed the `minDate` to include the day of the `minDate`, otherwise it would feel pretty odd. So using new Date() would include today